contracts/burner/tests/integration.rs

```diff
@@ -36,13 +38,12 @@ fn init_fails() {
     let msg = InitMsg {};
     let env = mock_env("creator", &coins(1000, "earth"));
     // we can just call .unwrap() to assert this was a success
-    let res: InitResult = init(&mut deps, env, msg);
-    match res.unwrap_err() {
-        StdError::GenericErr { msg, .. } => {
-            assert_eq!(msg, "You can only use this contract for migrations")
-        }
-        _ => panic!("expected migrate error message"),
-    }
+    let res: ContractResult<InitResponse> = init(&mut deps, env, msg);
+    let msg = res.unwrap_err();
+    assert_eq!(
+        msg,
+        "Generic error: You can only use this contract for migrations"
```

Comment:
Okay, we should add this to migration - simplify the tests (string compare) but different than the unit tests.